vignettes/Example1-Ostats-univariate.Rmd

```diff
@@ -220,13 +222,121 @@ Ostats_plot(plots = siteID,
             traits = trait, 
             overlap_dat = small_mammal_Ostats, 
             use_plots = sites2use, 
-            name_x = expression(paste('log'[10], ' body mass (g)')), 
+            name_x = 'log10 Body Mass (g)', 
```

Comment:
Please revert this change, I did that so that the plot we show in the journal would have a subscript for log10

Reply:
done